README.md

```diff
@@ -3,40 +3,68 @@ The repo for the terra data repository built by the jade team.
 
 See the DATABASE.md to set up the postgres database before you run jade.
 
-## Setup environment
-### Google project and account
+## Create kubernetes cluster
+
+In the google cloud console, within your personal project, go to Kubernetes Engine -> Clusters and create a cluster. In the default pool section change nodes to 1 and machine type to small. Open the Advanced section and click the checkbox to enable VPC native. Then click create.
+
+Once your cluster has finished creating, click the Connect button next to you cluster info. Copy the command and execute it on your local system. Now, if you click on docker -> kubernetes you should see a check next to the cluster you just created.
+
+
+## Create a service account
+In the google cloud console, go to IAM & Admin
```

Comment:
I _**really, really**_ appreciate this documentation!

On this step, what permissions does the service account need? What will it be used for?

Reply:
yes! This is amazing!